src/connector-cli/debugger/src/index.css

```
@@ -2,6 +2,211 @@
+  .dbg-input {
+    @apply block w-full h-10 px-md text-regular rounded bg-input-bg border border-input-border text-text-primary placeholder:text-text-placeholder outline-none shadow-none transition-colors;
+  }
+
+  .dbg-input:hover {
+    @apply border-input-focus;
+  }
+
+  .dbg-input:focus,
+  .dbg-input:focus-visible {
+    @apply outline-none border-input-focus;
+    box-shadow: 0 0 0 1px theme('colors.input.focus');
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)